src/core/client/admin/routes/Moderate/ModerateCard/ModerateCardContainer.tsx

```diff
@@ -81,6 +93,17 @@ class ModerateCardContainer extends React.Component<Props> {
     }
   };
 
+  private onShowCommentHistory = () => {
+    this.setState({
+      showHistory: true,
+    });
+  };
+  private onHideCommentHistory = () => {
+    this.setState({
+      showHistory: false,
+    });
+  };
+
```

Comment:
Replace with `useState`. :)

Reply:
Converted this class component to a function component. No more state.